### ISSUE TITLE
Exits without any pause for the user to speak

### ISSUE THREAD

**Issue by Mrs-Feathers** [zaf/asterisk-speech-recog#54]
i'm not sure what's going on [i've created and input my own API key and all that, but it seems to just error out every time with no debug messages.

```
    -- Executing [547@from-lynx:4] AGI("IAX2/lynx-iax-15602", "speech-recog.agi,en-US") in new stack
    -- Launched AGI Script /var/lib/asterisk/agi-bin/speech-recog.agi
    -- <IAX2/lynx-iax-15602>AGI Script speech-recog.agi completed, returning 0
```

this is the log of output. all it says is 0 for success. and there's no pause. it just returns 0 as soon as its run. no beep.

**Comment by jefaokpta** [comment]
Hi Katja,

Do you installed the dependencies?

On 13 Aug 2020, at 18:34, Katja DeCuir <notifications@github.com<mailto:notifications@github.com>> wrote:



i'm not sure what's going on [i've created and input my own API key and all that, but it seems to just error out every time with no debug messages.

    -- Executing [547@from-lynx:4] AGI("IAX2/lynx-iax-15602", "speech-recog.agi,en-US") in new stack
    -- Launched AGI Script /var/lib/asterisk/agi-bin/speech-recog.agi
    -- <IAX2/lynx-iax-15602>AGI Script speech-recog.agi completed, returning 0


this is the log of output. all it says is 0 for success.

**Comment by Mrs-Feathers** [comment]
Yes, as far as I can tell the dependencies are met. I am using Debian. So using apt rather than yum

**Comment by jefaokpta** [comment]
Im using Debian too, I’ve installed the following dependencies:


------------
Debian/Ubuntu dependencies
------------
Perl                    The Perl Programming Language
libio-socket-ssl-perl   Perl module implementing object oriented interface to SSL sockets
libwww-perl             simple and consistent interface to the world-wide web
libjson-perl            module for manipulating JSON-formatted data

asterisk-speech-recog<https://github.com/jefaokpta/asterisk-speech-recog>

I don’t changed anything in this repository.

On 14 Aug 2020, at 13:09, Katja DeCuir <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes, as far as I can tell the dependencies are met. I am using Debian. So using apt rather than yum

**Comment by Mrs-Feathers** [comment]
for some reason libjson-perl was not already marked as installed. i thought i had installed it manually. apparently not… and to my absolute amazement, its working!!!!!

    -- <PJSIP/500-00000002>AGI Script speech-recog.agi completed, returning 0
    -- Executing [508@from-internal:3] Verbose("PJSIP/500-00000002", "1,The text you just said is: Hello.") in new stack
 The text you just said is: Hello.
    -- Executing [508@from-internal:4] Verbose("PJSIP/500-00000002", "1,The probability to be right is: 0.83758235") in new stack

this is absolutely amazing.

----
73, KC1KSG

> Am 14.08.2020 um 15:44 schrieb Jefferson Alves <notifications@github.com>:
> 
> 
> Im using Debian too, I’ve installed the following dependencies: 
> 
> 
> ------------ 
> Debian/Ubuntu dependencies 
> ------------ 
> Perl The Perl Programming Language 
> libio-socket-ssl-perl Perl module implementing object oriented interface to SSL sockets 
> libwww-perl simple and consistent interface to the world-wide web 
> libjson-perl module for manipulating JSON-formatted data 
> 
> asterisk-speech-recog<https://github.com/jefaokpta/asterisk-speech-recog> 
> 
> I don’t changed anything in this repository. 
>
